### PR TITLE
docs: Update dynatrace-service documentation for merge

### PR DESCRIPTION
**Pull request by arthurpitman**

### CODE REVIEW

File: content/docs/0.9.x/monitoring/dynatrace/install/index.md [hunk 1]
```diff
@@ -1,90 +1,110 @@
 ---
 title: Install
-description: Setup Dynatrace monitoring and Keptn integration
+description: Set up Dynatrace monitoring and Keptn integration
 weight: 1
 icon: setup
 ---
 
-## Deploy Dynatrace Operator on Kubernetes
+## Deploy Dynatrace OneAgent Operator on Kubernetes
```
Comment (heckelmann) [inline content/docs/0.9.x/monitoring/dynatrace/install/index.md:8]: Not sure cause the official name is Dynatrace Operator, while the deprecated/old version was called Dynatrace OneAgent Operator. I would use Dynatrace Operator
image

Comment (arthurpitman) [reply content/docs/0.9.x/monitoring/dynatrace/install/index.md]: Updated to Dynatrace Operator

**Review by j-poecher — CHANGES_REQUESTED** [review]

File: content/docs/0.9.x/monitoring/dynatrace/configure_slis/index.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+```yaml
+---
+spec_version: '1.0'
+indicators:
+  throughput: "builtin:service.requestCount.total:merge(0):count?scope=tag(keptn_project:$PROJECT),tag(keptn_stage:$STAGE),tag(keptn_service:$SERVICE),tag(keptn_deployment:$DEPLOYMENT)"
+  error_rate: "builtin:service.errors.total.count:merge(0):avg?scope=tag(keptn_project:$PROJECT),tag(keptn_stage:$STAGE),tag(keptn_service:$SERVICE),tag(keptn_deployment:$DEPLOYMENT)"
+  response_time_p50: "builtin:service.response.time:merge(0):percentile(50)?scope=tag(keptn_project:$PROJECT),tag(keptn_stage:$STAGE),tag(keptn_service:$SERVICE),tag(keptn_deployment:$DEPLOYMENT)"
+  response_time_p90: "builtin:service.response.time:merge(0):percentile(90)?scope=tag(keptn_project:$PROJECT),tag(keptn_stage:$STAGE),tag(keptn_service:$SERVICE),tag(keptn_deployment:$DEPLOYMENT)"
+  response_time_p95: "builtin:service.response.time:merge(0):percentile(95)?scope=tag(keptn_project:$PROJECT),tag(keptn_stage:$STAGE),tag(keptn_service:$SERVICE),tag(keptn_deployment:$DEPLOYMENT)"
```
[the remark this answers is not in the document]
Comment (arthurpitman) [reply content/docs/0.9.x/monitoring/dynatrace/configure_slis/index.md]: Fixed

**Review by j-poecher — APPROVED** [review]
lgtm

**Comment by johannes-b** [comment]
@arthurpitman from a content point of view it looks good to me!

- [ ] But can you please update the page: https://deploy-preview-922--keptn.netlify.app/docs/integrations/ (similar to what we did with the prometheus-service. Please see the **Note:** below the tables.)

We are in a very unlucky situation because the Keptn team is already working on the docs for 0.10. as you can see on this PR: https://github.com/keptn/keptn.github.io/pull/920
In this PR, a copy of the folder `0.9.x` has been made and renamed to `0.10.x`. After merging your PR, we need to make sure that a second PR provides the same content/changes also for the `0.10.x` folder. Can you please take care of the second PR?

**Comment by arthurpitman** [comment]
- Updates made to `docs/integrations/` (similar to the prometheus-service)
- A second PR, #928, has been created to update the 0.10.x docs

**Review by Kavindu-Dodan — APPROVED** [review]

**Review by johannes-b — APPROVED** [review]
lgtm

**Pull request merged**